packages/suite/src/views/dashboard/components/AssetsCard/components/AssetGrid/index.tsx

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+import styled, { useTheme } from 'styled-components';
```

Comment:
super nitpick, but it may be more future-proof to import useTheme from `@trezor/components`, since we are working towards having reusable UI components working on react-native in the future and `useTheme` from our UI lib already has some initial work done.

Reply:
Nice nitpick. I am excited for react-native app! ...Done